recipes-core/searcher-container/files/searcher-network-init

```diff
@@ -166,11 +170,10 @@ start_firewall() {
     ###########################################################################
     # (7) ALWAYS_ON_OUT: Outbound rules that never turn off
     ###########################################################################
-    # CL P2P outbound on port 9000 (TCP + UDP)
-    # Note: This is the lighthouse CL client run on the host, not the searcher's CL
-    $IPTABLES -A $CHAIN_ALWAYS_ON_OUT -p tcp --dport $CL_P2P_PORT \
+    # Sync proxy outbound on port 8552 (TCP only) - IP whitelisted
+    $IPTABLES -A $CHAIN_ALWAYS_ON_OUT -p tcp -d $FLASHBOTS_NGINX_IP_1 --dport $SYNC_PROXY_PORT \
         -m conntrack --ctstate NEW -j ACCEPT
-    $IPTABLES -A $CHAIN_ALWAYS_ON_OUT -p udp --dport $CL_P2P_PORT \
+    $IPTABLES -A $CHAIN_ALWAYS_ON_OUT -p tcp -d $FLASHBOTS_NGINX_IP_2 --dport $SYNC_PROXY_PORT \
         -m conntrack --ctstate NEW -j ACCEPT
```

Comment:
Ah, I see that above we have
```
$IPTABLES -A OUTPUT -m conntrack --ctstate ESTABLISHED,RELATED -j ACCEPT
```

In that case, these outbound rules are not required and can safely be removed